docs/guides/_include/zbm-setup.rst

```diff
@@ -11,21 +11,18 @@ Set ZFSBootMenu properties on datasets
       Assign command-line arguments to be used when booting the final kernel. Because ZFS properties are inherited,
       assign the common properties to the ``ROOT`` dataset so all children will inherit common arguments by default.
 
-      .. parsed-literal::
-
-        zfs set org.zfsbootmenu:commandline="|zbmkcl|" zroot/ROOT
+      .. include:: _include/commandline.rst
 
   .. group-tab:: Encrypted
 
       Assign command-line arguments to be used when booting the final kernel. Because ZFS properties are inherited,
       assign the common properties to the ``ROOT`` dataset so all children will inherit common arguments by default.
 
-      .. parsed-literal::
-
-        zfs set org.zfsbootmenu:commandline="|zbmkcl|" zroot/ROOT
+      .. include:: _include/commandline.rst
 
       Setup key caching in ZFSBootMenu.
 
-      .. parsed-literal::
+      .. code-block::
 
-        zfs set org.zfsbootmenu:keysource="zroot/ROOT/\ |distribution|" zroot
+        source /etc/os-release
```

Comment:
This step *might* be unnecessary if ID is exported when os-release is first sourced. See associated edits in that section.

Reply:
agreed, might as well export ID and call it good